### PR TITLE
Reduce Pool1 Swap Fee via Updgrade Handler

### PR DESCRIPTION
**Pull request by mattverse**
Closes: #[1424](https://github.com/osmosis-labs/osmosis/issues/1424)

## What is the purpose of the change

This PR originates from prop 214, where via governance, decided to reduce Pool1(ATOM/OSMO) swap fee from 0.3% to 0.2%. The logic for this is under v8 Upgrade handler.

## Brief Changelog 
  - Changes pool swap fee from 0.3% to 0.2%

## Testing and Verifying

This change added tests and can be verified as follows:

Added unit test that validates swap fee change.
## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? no
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? no
  - How is the feature or change documented? not documented

### CODE REVIEW

File: app/upgrades/v8/prop214.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package v8
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	gammkeeper "github.com/osmosis-labs/osmosis/v7/x/gamm/keeper"
+	"github.com/osmosis-labs/osmosis/v7/x/gamm/pool-models/balancer"
+)
+
+func Prop214(ctx sdk.Context, gamm *gammkeeper.Keeper) {
```
Comment (ValarDragon) [inline app/upgrades/v8/prop214.go:10]: Lets link to mintscan prop

**Review by ValarDragon — APPROVED** [review]

File: app/upgrades/v8/prop214.go (hunk 1 above)
Comment (alexanderbez) [reply app/upgrades/v8/prop214.go:10]: Good idea.

**Comment by ValarDragon** [comment]
I applied comment suggestions

**Comment by mattverse** [comment]
tysm!

**Pull request merged**